src/components/measureGroups/MeasureGroups.test.tsx

```diff
@@ -399,21 +399,6 @@ describe("Measure Groups Page", () => {
     userEvent.click(
       screen.getByTestId("leftPanelMeasureInformation-MeasureGroup1")
     );
```

Comment:
Are we expecting any actions after the click event, if not is it required to trigger a click event?

Reply:
Sorry, I forgot to remove it. I'll remove it and will push it